include/powsybl/iidm/ReactiveLimitsHolder.hpp

```diff
@@ -21,6 +21,8 @@ class ReactiveLimits;
 
 class ReactiveLimitsHolder {
 public:
+    explicit ReactiveLimitsHolder(std::unique_ptr<ReactiveLimits>&& limits);
```

Comment:
`DanglingLineGeneration` inherits from `ReactiveLimitsHolder` so I thought it was better to call to this constructor (**which should be protected**) instead of calling `setReactiveLimits` within constructor body (which is more c++ friendly).